.gitignore

```diff
@@ -0,0 +1,6 @@
+### macOS ###
+## Apple Finder's file
```

Comment:
nit:

you emulate headings so it should be inverted: macOS heading is "bigger" than "nested" headings (remember markdown syntax?):

```suggestion
## macOS ###
### Apple Finder's file
```

Reply:
My bad, fixed with https://github.com/ios-course/ironfoudation-team-project/pull/24/commits/9749257c6b99d25724eb9730fca446b36ecf3ca3.